frontend/__tests__/unit/pages/About.test.tsx

```
@@ -554,19 +558,14 @@ describe('About Component', () => {
-          error: null,
-        }
+    ;(useQuery as unknown as jest.Mock).mockImplementation((query, options) => {
+      if (options?.variables?.key === 'nest') {
+        return { loading: false, data: { project: null }, error: null }
+      }
+      return {
+        loading: false,
+        data: { user: { avatarUrl: '', company: '', name: 'Dummy', location: '' } },
+        error: null,
       }
       return { loading: true }
```

Biome (2.3.13)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)